plugin/src/main/java/io/fabric8/maven/plugin/ResourceMojo.java

```diff
@@ -453,6 +472,26 @@ private KubernetesList convertToOpenShiftResources(KubernetesList resources) {
                 }
             }
         }
+        if (isOpenShiftMode()) {
+            for (ImageConfiguration image : this.resolvedImages) {
+                String name = image.getName();
+                int idx = name.lastIndexOf(':');
+                if (idx < 0) {
+                    log.warn("No ':' in image name: " + name);
```

Comment:
if we do a s2i build and can't find a tag it means we can't ever deploy that s2I image stream into any other namespace ever. So its  pretty serious issue. We get a tag even for using snapshot / latest.